packages/esm-commons-lib/package.json

```diff
@@ -32,23 +32,21 @@
   },
   "dependencies": {
     "@carbon/react": "^1.13.0",
+    "@openmrs/openmrs-form-engine-lib": "next",
```

Comment:
@pirupius why the `esm-commons-lib` have to depend on `the form-engine-lib`

Reply:
I see you removed from the devDependencies but I still don't understand.

Reply:
The encounter list component opens the form with the form-engine-lib

Reply:
About us removing it from devDependencies. We agreed during Friday scrum to make it a dependency to fix issues where the versions of the form engine were not aligned.